pkg/apis/config/spire_config_test.go

```diff
@@ -0,0 +1,72 @@
+/*
+Copyright 2021 The Tekton Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package config_test
+
+import (
+	"testing"
+
+	"github.com/google/go-cmp/cmp"
+	"github.com/tektoncd/pipeline/pkg/apis/config"
+	test "github.com/tektoncd/pipeline/pkg/reconciler/testing"
+	sc "github.com/tektoncd/pipeline/pkg/spire/config"
+	"github.com/tektoncd/pipeline/test/diff"
+)
+
+func TestNewSpireConfigFromConfigMap(t *testing.T) {
+	type testCase struct {
+		expectedConfig *sc.SpireConfig
+		fileName       string
+	}
+
+	testCases := []testCase{
+		{
+			expectedConfig: &sc.SpireConfig{
+				TrustDomain:     "test.com",
+				SocketPath:      "unix:///spiffe-workload-api/test-spire-agent.sock",
```

Comment:
Reference the defined `const` values instead of repeating the strings, or use bogus values that are clearly test values like `unix:///somewhere.sock`.

Reply:
Done.